data_browser/src/ExperimentPage.js

```diff
@@ -312,14 +312,15 @@ function renderTrainLmStep({step, steps}) {
   // Model
   const modelConfig = trainConfig.model;
   const architectureSummary = [
+    modelConfig.n_routed_experts && `${modelConfig.n_routed_experts} experts (${modelConfig.n_shared_experts} shared, ${modelConfig.num_experts_per_tok} active)`,
```

Comment:
The total number of experts should be `n_routed_experts + n_shared_experts`. where `n_shared_experts` are shared and `num_experts_per_tok` experts are chosen from `n_routed experts`. I think it's a bit unclear from your description here.